src/buza/settings_env_dev.py

```diff
@@ -27,4 +27,7 @@
     str(checkout_dir.joinpath('node_modules')),
 ]
 
+# Include the local host by default for development.
+ALLOWED_HOSTS = ['127.0.0.1', 'localhost']
```

Comment:
@pjdelport do we still need this then?

Reply:
It's not a bad idea to keep: it's not used when `DEBUG` is enabled, but it's useful to have for testing / developing with `DEBUG` off (such as when testing the 404 template).